packages/components/src/FilterDateRangePicker/FilterDateRangePicker.tsx

```diff
@@ -30,12 +31,14 @@ export const FilterDateRangePicker = ({
     setIsOpen={setIsOpen}
     renderTrigger={(triggerProps): JSX.Element =>
       renderTrigger({
-        selectedValue: isCompleteDateRange(selectedRange) ? (
-          <DateRangeDisplayLabel
-            dateRange={selectedRange}
-            locale={getLocale(locale)}
-          />
-        ) : undefined,
+        selectedValue:
+          isCompleteDateRange(selectedRange) &&
```

Comment:
We can probably remove this now right? Given `isValidRange` also checks that both values exist.

Reply:
It's weird cause TS throws an error on `dateRange={selectedRange}` cause it could still be undefined.

Reply:
So...this?

```suggestion
          selectedRange &&
```

Reply:
Also no bueno lol
![image](https://github.com/cultureamp/kaizen-design-system/assets/820635/a09d42f7-bc95-47d4-a904-041864cca9fe)

Reply:
Ohhhhhh! Because `isCompleteDateRange` is a type guard haha. I guess it's not exactly the same as the validity check 🤔 Okay, keep it I guess lol.